hbase-client/src/main/java/org/apache/hadoop/hbase/NotAllRootRegionsOnlineException.java

```diff
@@ -25,19 +25,19 @@
  * Thrown when an operation requires the root and all meta regions to be online
  */
 @InterfaceAudience.Public
-public class NotAllMetaRegionsOnlineException extends DoNotRetryIOException {
+public class NotAllRootRegionsOnlineException extends DoNotRetryIOException {
```

Comment:
How many ROOT regions? If one, shouldn't be plural? Or 'All'?

Reply:
Ah right, I just compied meta which in hindsight did make sense for meta and not for root will change :-)